lib/api/LogManagerImpl.hpp

```diff
@@ -340,10 +347,20 @@ namespace MAT_NS_BEGIN
         DataViewerCollection m_dataViewerCollection;
         std::vector<std::shared_ptr<IDataInspector>> m_dataInspectors;
         std::recursive_mutex m_dataInspectorGuard;
+
+        std::mutex m_pause_mutex;
+        std::condition_variable m_pause_cv;
+        uint64_t m_pause_active_count = 0;
+        enum class PauseState : uint8_t
+        {
+            Active,
```

Comment:
Can we call this `NotPaused`? To me, `PauseState::Active` reads as we're actively paused.